package.json

```diff
@@ -81,7 +81,8 @@
                         "bat": "",
                         "shellscript": "bash",
                         "fsharp": "fsi",
-                        "csharp": "scriptcs"
+                        "csharp": "scriptcs",
+                        "vbscript": "cscript //Nologo"
```

Comment:
I noticed .vbs uses Visual Basic by default. I had to change the language mode to VBScript.

How would changing the code-runner to use 'vb' instead of 'vbscript' affect any of the .vb files for vb.net?